### PR TITLE
[MBL-859] Add deprecation banner for creator dashboard

### PR DESCRIPTION
**Pull request by mtgriego**
# 📲 What

Adds a deprecation banner for the creator dashboard

Please note we are only using the "After August 14..." string, the other untranslated string is being used by iOS

# 🤔 Why

We will eventually be removing this feature so we wanted to warn the users

# 🛠 How

Added a compose view and strings at the top of the page

# 👀 See

https://github.com/kickstarter/android-oss/assets/7563288/91978821-97fd-4058-9a8d-0f93d83bc360

# 📋 QA

Go to the creator dashboard page and see that this is presented

# Story 📖

[MBL-859](https://kickstarter.atlassian.net/browse/MBL-859)

[MBL-859]: https://kickstarter.atlassian.net/browse/MBL-859?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/src/main/res/values-de/strings_i18n.xml [hunk 1]
```diff
@@ -16,6 +16,7 @@
   <string name="Add_new_card" formatted="false">Kreditkarte hinzufügen</string>
   <string name="Add_ons" formatted="false">Add-ons</string>
   <string name="Add_ons_unavailable" formatted="false">Add-ons nicht verfügbar</string>
+  <string name="After_August_14_2023_this_Dashboard_feature_will_only_be_available_on_our_website" formatted="false">After August 14, 2023, this Dashboard feature will only be available on our website.</string>
```
Comment (mtgriego) [inline app/src/main/res/values-de/strings_i18n.xml:19]: Working with translation team to get this fixed

File: app/src/main/java/com/kickstarter/ui/compose/KSComposeColor.kt [hunk 2]
```diff
@@ -13,3 +13,4 @@ val kds_support_500 = Color(0xFF464646)
 val kds_support_700 = Color(0xFF222222)
 val kds_white = Color(0xFFFFFFFF)
 val kds_black = Color(0xff000000)
+val kds_alert = Color(0xFFA12027)
```
Comment (Arkariang) [inline app/src/main/java/com/kickstarter/ui/compose/KSComposeColor.kt:16]: ❤️

File: app/src/main/java/com/kickstarter/ui/compose/KSImageTextBanner.kt [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    KSImageTextBanner(
+        imageResToDisplay = R.drawable.ic_alert,
+        textResToDisplay = R.string.After_August_14_2023_this_Dashboard_feature_will_only_be_available_on_our_website,
+        textColorRes = R.color.kds_white,
+        backgroundColorRes = R.color.kds_alert
+    )
+}
+
+@Composable
+fun KSImageTextBanner(
```
Comment (Arkariang) [inline app/src/main/java/com/kickstarter/ui/compose/KSImageTextBanner.kt:33]: 💖

**Comment by Arkariang** [comment]
@mtgriego we could add the `/ui/compose` directory to be ignored by the code coverage at `jacoco.gradle`
Screenshot 2023-06-27 at 3 55 21 PM

**Review by Arkariang — APPROVED** [review]

**Review by Arkariang — APPROVED** [review]

**Pull request merged**